### ISSUE TITLE
Use mdoc for the microsite

### ISSUE THREAD

**Issue by gvolpe** [profunktor/http4s-tracer#158]
`tut` is no longer maintained and has issues with new versions of `sbt-microsites`.

**Comment by asachdeva** [comment]
Would love to work on this ticket if still available.

**Comment by gvolpe** [comment]
Please do @asachdeva ! You can have a look at https://github.com/profunktor/http4s-jwt-auth/pull/42 for reference :)

**Comment by asachdeva** [comment]
Hey @gvolpe I have started the work using your cited example in this branch https://github.com/asachdeva/http4s-tracer/tree/feature/158_MDoc

The sbt mdoc target currently barfs with 12 errors in the guide.md documentation.  I will try and see how to fix this and I tried toggling the scalacOptions in mdoc as well but was unable to get past it.  Let me know if you have any ideas other than fixing the "broken" documentation itself.
